pkg/config/logconfig.go

```diff
@@ -43,13 +45,20 @@ func (l *LogConfiguration) setupLogger(agentType AgentType) error {
 		MaxMetricSize(l.MetricFile.MaxSize).
 		MaxMetricBackups(l.MetricFile.MaxBackups).
 		MaxMetricAge(l.MetricFile.MaxAge).
+		Usage(agentType == TraceabilityAgent && l.UsageFile.Enabled).
+		UsageFilename(l.UsageFile.Name).
+		MaxPublishedTransactionAge(l.UsageFile.MaxAge).
```

Comment:
I think we may need to add all the rotate properties here. The way the options here are used will not map specifically to how the story lays it out. But as long as we allow them to tune the settings they can do what they want.